packages/ui/src/index.tsx

```diff
@@ -18,7 +18,7 @@ const uiConfig = JSON.parse(
   document.getElementById('__UI_CONFIG__')?.textContent || '{}'
 ) as UIConfig;
 
-initI18n({ lng: uiConfig.locale?.lng || 'en-US', basePath }).then(() => {
+initI18n({ lng: uiConfig.locale?.lng || navigator.language || 'en-US', basePath }).then(() => {
```

Comment:
Is it possible to use the settings value first, then the global ui-config?

Reply:
tried, but can't use hooks here

Reply:
How about using the store it self (I'm not sure if zustand supports that)